spec/presenters/car_presenter_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CarPresenter do
+  context 'contain fields' do
+    %i[manufacturer model year color license_plat_number].each do |field|
+      it { expect(CarPresenter::MODEL_ATTRIBUTES).to include field }
+    end
+  end
```

Comment:
1. Why do you iterate an array to compare fields?
2. Test `delegate(*MODEL_ATTRIBUTES, to: :record)` like in a example below:

```ruby
describe 'constants' do
  describe 'MODEL_ATTRIBUTES' do
    subject { described_class::MODEL_ATTRIBUTES }
   
    it { is_expected.to eq %i[manufacturer model year color license_plat_number] }
  end
end

describe 'delegation' do
  it { is_expected.to delegate_method(:id).to(:record) }
  # all fields which is delegated
end
```